build.proj

```diff
@@ -531,6 +531,9 @@
   <!-- Run checkin tests for each pull request -->
   <Target Name="Test" DependsOnTargets="BeforeRunTests">
     <Message Importance="high" Text="Running check in tests..." />
+    <!-- Run Pester tests -->
```

Comment:
Let's set this to 'ErrorAndContinue' so that we find any other errors in the PR as well, but still fail if this fails, see: https://msdn.microsoft.com/en-us/library/7z253716.aspx

Reply:
done